CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ list(APPEND CMAKE_MODULE_PATH ${CORENEURON_PROJECT_SOURCE_DIR}/CMake
 # HPC Coding Conventions
 # =============================================================================
 set(CODING_CONV_PREFIX "CORENRN")
+# Disable sort-includes for now, because headers are not clean
```

Comment:
An alternative is to add the file `.clang-format.changes` to the repository with the following change:
```yaml
SortIncludes: false
```

Reply:
Tristan's suggestion would be better!